Santhel

Santhel opened the pull request:
The first assignments of Santhel

s-will commented:
Hi @Santhel 
your submissions need a few changes to activate the automatic "continuous integration" (CI)  test of your programs by Travis. I introduced the following changes to activate the test for your A0 submission and get you started with my automatic testing system.
 * registered you as participant (see change of file participants.yml)
 * renamed your program from A0/A0.py to A0/Santhel-HelloWorld.py
 * registered your submission (see file submissions.yml)

Please register your other submissions analogously (feel free to ask if you are unsure how). Then, check the report of Travis CI (click on Details after "continuous-integration/travis-ci/pr") and fix the programs where necessary. (For example, your A0 submission prints an additional empty line, which must be suppressed.)

Santhel commented:
@s-will I added my data in the submissions.yml in the Santhel-branch. I also changed the output of my Hello-World-Program so that it would not print the newline. I don't know what else I have to do.
Travis seems to assume Python3 for testing so my programs in Python2.7 don't work.
Do all tests have to be fulfilled to pass?

Santhel commented:
@s-will You gave me time until end of July, I'm sorry I could not finish everything in time (mainly correcting Santhel-Administration and programming k-letShuffle)
I did not expect that the running through travis would be mandatory, since my programs do not fulfill the output-file-requirements (I did not consider them), some programs might give an entirely different output (e.g. Santhel-Hamster since I had an other understanding of the algorithm).
Is there still something I can do or is that it and the grades for this course will be entered. If so, I would like to go over my programs in person (or via e-mail) with you.

s-will commented:
Hi @Santhel, don't worry. I very much appreciate your effort to keep up and submit all these programs in the agreed time. Thus, certainly, I would not assign a bad grade only because CI tests are not passing. I need to find some time to look at your submissions first. If changes are necessary, I will let you know. In any case, I will come back to you before I decide on a grade.

PS: I fixed the language issue; specifying language python2 should give you python 2.7

s-will commented:
After my change of the submission language. there seems to be a problem with the CI submission system. I'll try to fix this later
